RELEASE/scripts/autoscend/paths/low_key_summer.ash

```diff
@@ -372,6 +366,14 @@ boolean LX_lowkeySummer() {
 		if (L5_getEncryptionKey() || L5_findKnob()) { return true; }
 	}
 
+	// If we have the resources to do the Haunted Kitchen in the minimum adventures, we should do it sooner 
+	if (internalQuestStatus("questM20Necklace") == 0) {
+		backupSetting("auto_delayHauntedKitchen", true);
+		boolean advSpent = LX_unlockHauntedBilliardsRoom();
+		restoreSetting("auto_delayHauntedKitchen");
```

Comment:
Would it maybe be better to use `LX_unlockHauntedBilliardsRoom(boolean ignore_delay)`
with `LX_unlockHauntedBilliardsRoom()` always using (false) but in this case it sends (true)?
I am asking because I am not 100% sure myself

Reply:
That's a good idea. I shall implement and update.